src/MediaInfoBundle/Resources/views/Download/windowsTable.html.twig

```diff
@@ -13,11 +13,16 @@
     <td>If you don't know which one to download, download this one</td>
 </tr>
 <tr>
-    <th rowspan="3">64 bit</th>
+    <th rowspan="4">64 bit</th>
     <th><abbr title="Graphical User Interface">GUI</abbr></th>
     <td><a href="{{ downloadLink }}">v21.09 Universal (32/64 bit)</a> <small>(<a href="//mediaarea.net/download/binary/mediainfo-gui/21.09/MediaInfo_GUI_21.09_Windows_x64_WithoutInstaller.7z">64 bit only without installer</a>)</small></td>
     <td></td>
 </tr>
+<tr>
+    <th>Graph vizualisation plugin for the GUI</th>
```

Comment:
String too long and typo.
`<abbr title="Graph visualization plugin for the GUI">Graph</abbr>`

BTW, for the comment part: `Manual download in case the automatic install from the GUI does not work.`

(2x)